telemetry/definitions/commonDefinitions.json

```diff
@@ -1825,6 +1830,31 @@
                     "type": "codewhispererTriggerType"
                 }
             ]
+        },
+        {
+            "name": "codewhisperer_perceivedLatency",
```

Comment:
Instead of creating a new definition, would it be possible to add `codewhispererPerceivedLatencyDuration` to some other event? `codewhisperer_userDecision` seems like a logical choice as it's only possible to pick a suggestion if you can see it. Or at least I would assume that's the case.

Reply:
@JadenSimon `codewhisperer_userDecision` will be emitted for every recommendation(even the ones are not seen by users), but this metrics will only be emitted at most once for every `serviceInvocation` event, only if the first recommendation was shown to the user. So it's better for it to be separate.

Reply:
If this way is easier for y'all then it works for me. Just make sure it's emitted around the same time as `codewhisperer_userDecision` and marked passive.